src/theft_rng.h

```diff
@@ -3,28 +3,24 @@
 
 #include <stdint.h>
 
-/* Wrapper for Mersenne Twister.
- * See copyright and license in theft_rng.c, more details at:
- *     http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/emt.html
- *
- * Local modifications are described in theft_mt.c. */
+/* Wrapper for xoroshiro128+. */
 
-/* Opaque type for a Mersenne Twister PRNG. */
+/* Opaque type for the state of an xoroshiro128+ PRNG. */
```

Comment:
Probably better to just say "a PRNG instance", while it's changing anyway. The comment above already specifies the specific PRNG used.